clients/src/main/resources/common/message/DeleteTopicsRequest.json

```diff
@@ -23,10 +23,17 @@
   //
   // Version 5 adds ErrorMessage in the response and may return a THROTTLING_QUOTA_EXCEEDED error
   // in the response if the topics deletion is throttled (KIP-599).
-  "validVersions": "0-5",
+  //
+  // Version 6 reorganizes topics, adds topic IDs and allows topic names to be null.
+  "validVersions": "0-6",
   "flexibleVersions": "4+",
   "fields": [
-    { "name": "TopicNames", "type": "[]string", "versions": "0+", "entityType": "topicName",
+    { "name": "Topics", "type": "[]DeleteTopicState", "versions": "6+", "about": "The name or topic ID of the topic",
+      "fields": [
+        {"name": "Name", "type": "string", "versions": "6+", "nullableVersions": "6+", "about": "The topic name"},
```

Comment:
Why is topic name nullable but topicId not? do you mean we will fill all delete requests with a topicId before sending a request?

Reply:
We will use the zero UUID to represent null as described in the KIP.